packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -237,10 +237,16 @@ export const getVideoReceiverMqa = ({
   const lastPacketsLost = getLastTotalValue('totalPacketsLost');
   const lastBytesReceived = getLastTotalValue('totalBytesReceived');
 
+  const lastRtxPacketsReceived = getLastTotalValue('retransmittedPacketsReceived');
+  const lastRtxBytesReceived = getLastTotalValue('retransmittedBytesReceived');
+
   const packetsLost = getTotalValue('totalPacketsLost');
   const totalPacketsReceived = getTotalValue('totalPacketsReceived');
   const totalBytesReceived = getTotalValue('totalBytesReceived');
 
+  const totalRtxPacketsReceived = getTotalValue('retransmittedPacketsReceived');
+  const totalRtxBytesReceived = getTotalValue('retransmittedBytesReceived');
```

Comment:
If we receive multiple streams, does this give us the total number of RTX packets/bytes received across all the streams?

Reply:
I did some digging regarding this, and it seems to be working as expected. I performed a manual test to compare the results between two MQAs, and here are my findings:

This is the first MQA, which shows the `rtxPackets` as `35`. The `35` is probably from the previous MQAInterval.

![Screenshot 2024-06-26 at 4 54 58 PM](https://github.com/webex/webex-js-sdk/assets/67634227/8ce0ac6d-4a27-4931-b1b1-e04ce0020299)

And this is the log from `result.retransmittedPacketsReceived` and `result.retransmittedBytesReceived` just before the MQE was sent.

![Screenshot 2024-06-26 at 4 55 12 PM](https://github.com/webex/webex-js-sdk/assets/67634227/06fea21c-59f1-4d37-a4ac-61d7b8dbe614)

The upcoming MQA shows the `rtxPackets` to be `81`.

![Screenshot 2024-06-26 at 4 55 39 PM](https://github.com/webex/webex-js-sdk/assets/67634227/345afddc-d269-428e-b892-99f6d98fcb7c)

And here is the log from `result.retransmittedPacketsReceived` and `result.retransmittedBytesReceived` just before the MQA was sent.

![Screenshot 2024-06-26 at 4 55 32 PM](https://github.com/webex/webex-js-sdk/assets/67634227/f6102153-7882-41c7-88dc-f824a30cc572)

From the last MQA, we have `188` (SSRC `3839358778`) and `676` (SSRC `2511649477`). From the previous MQA, for the same SSRCs, we have `175` and `608`.

Subtracting the values from the previous MQA from the last MQA, we get that `13` packets from SSRC `3839358778` and `68` packets from SSRC `2511649477` were retransmitted, summing up to `81` retransmitted packets within the last MQAInterval, which corresponds to the value from the last MQA interval.

Reply:
Ah thanks for looking into it. I do see now that we have a util function `getTotalValueFromBaseType` that helps with this. I'll go ahead and approve your PR.